app/controllers/foo_controller.rb

```diff
@@ -0,0 +1,13 @@
+class FooController < ActionController::Base  
+  def new_user
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
```

Comment:
__[development][rubocop]__
* Lint/UselessAssignment: Useless assignment to variable - `user`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)
* Style/RedundantParentheses: Don't use parentheses around a literal.